astropy/stats/__init__.py

```diff
@@ -11,14 +11,35 @@
 
 """
 
+from . import funcs
 from .funcs import *  # noqa
+from . import biweight
 from .biweight import *  # noqa
+from . import sigma_clipping
 from .sigma_clipping import *  # noqa
+from . import jackknife
 from .jackknife import *  # noqa
+from . import circstats
 from .circstats import *  # noqa
+from . import bayesian_blocks as _bb
 from .bayesian_blocks import *  # noqa
+from . import histogram as _hist
 from .histogram import *  # noqa
+from . import info_theory
 from .info_theory import *  # noqa
-from .lombscargle import *  # noqa
+from . import spatial
 from .spatial import *  # noqa
+from .lombscargle import *  # noqa
 from .bls import *  # noqa
+
+
```

Comment:
Okay, thanks for the clarification! That makes sense. Just a minor comment left and should be all good.
```suggestion
# This is to avoid importing deprecated modules in subpackage star import
```